src/backend/distributed/deparser/deparse_text_search.c

```diff
@@ -253,6 +364,29 @@ DeparseAlterTextSearchConfigurationStmt(Node *node)
 }
 
 
+/*
+ * DeparseAlterTextSearchConfigurationStmt returns the sql representation of any generic
+ * ALTER TEXT SEARCH DICTIONARY .... statement. The statements supported include
+ * - ALTER TEXT SEARCH DICTIONARY name ( option [ = value ] [, ... ] )
+ */
+char *
+DeparseAlterTextSearchDictionaryStmt(Node *node)
+{
+	AlterTSDictionaryStmt *stmt = castNode(AlterTSDictionaryStmt, node);
+
+	StringInfoData buf = { 0 };
+	initStringInfo(&buf);
+
+	char *identifier = NameListToQuotedString(castNode(List, stmt->dictname));
+	appendStringInfo(&buf, "ALTER TEXT SEARCH DICTIONARY %s ( ", identifier);
+
+	char *objectName = "DICTIONARY";
+	AppendDefElemListInternal(&buf, stmt->options, objectName);
```

Comment:
shouldn't/couldn't this be using `AppendDefElemListForDictionary`?
if there is a good reason for _not_ using that function, would it make more sense to remove the specialized functions and call them everywhere like this?

Maybe even simpler like;
```suggestion
	AppendDefElemListInternal(&buf, stmt->options, "DICTIONARY");
```
As I don't see much use in first assigning it to a variable.